packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -323,6 +324,20 @@ NOTE: this is ${styleBold.wrap('not')} recommended. Asset changes cannot be incl
     );
   }
 
+  void assertReleaseContainsPlatform({
+    required Release release,
+    required Patcher patcher,
+  }) {
+    final containsPlatform = release.platformStatuses
+        .containsKey(patcher.releaseType.releasePlatform);
+    if (!containsPlatform) {
+      logger.err(
+        '''No release exists for [platform]. Please run shorebird release [platform] to create one.''',
```

Comment:
Why not print the platform instead of `[platform]`?

Reply:
Sounds good! I took this message from the issue. And I just realized the [platform] should be interpolated. Let me add a better one

Reply:
Changed for `No release exists for android in 1.1.0. Please run shorebird release --platform android to create one.`

Let me know what you think